test/jdk/java/awt/Robot/HiDPIScreenCapture/ScreenCaptureGtkTest.java

```diff
@@ -76,7 +81,7 @@ public void paint(Graphics g) {
                 // in order to test color picker's precision.
                 for (int i = 1; i < 4; i++) {
                     g.setColor(COLORS[i]);
-                    g.fillRect(i, 0, 1, 1);
+                    g.fillRect(leftOffset + i, topOffset, 1, 1);
```

Comment:
May I suggest using `COLORS.length` instead of 4 in the for-loops for painting and checking?

Reply:
Sure. Will fix that.